jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet

```diff
@@ -89,7 +89,19 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           '--path.procfs=/host/proc',
           '--path.sysfs=/host/sys',
           '--path.rootfs=/host/root',
-
+          '--no-collector.wifi',
+          '--no-collector.hwmon',
+          '--no-collector.netstat',
+          '--no-collector.sockstat',
+          '--no-collector.entropy',
+          '--no-collector.filefd',
+          '--no-collector.time',
+          '--no-collector.conntrack',
+          '--no-collector.bcache',
+          '--no-collector.stat',
+          '--no-collector.xfs',
+          '--no-collector.pressure',
```

Comment:
It is providing CPU and memory pressure mechanism information and I am not sure if those are not reflecting k8s scheduler states.